Add viaIR option to CompilerStack (and solc) (#10147)
ethereum/solidity: Changelog.md

### 0.7.5 (unreleased)

Compiler Features:

>  * Command Line Interface: New option ``--experimental-via-ir`` allows examining generated code via the Yul intermediate representation.

Reviewer: "examining generated code" sounds like it would do the same as ``--ir``.

Author: Fixed.

### 0.7.5 (unreleased)

Compiler Features:
 * Command Line Interface: New option ``--experimental-via-ir`` allows switching compilation process to go through

>    via the Yul intermediate representation. This is highly experimental and there for development purposes.

Reviewer 2: ```suggestion
   the Yul intermediate representation. This is highly experimental and is used for development purposes.
```
